### PR TITLE
Ops 682/task 1349

### PR DESCRIPTION
**Pull request by fpigeonjr**
## What changed

Adds a high-level `TablePageLayout` component and replaces the Agreements list page with it.
- does not include Summary Card section

## Issue

#1349 

## How to test

1. Goto the Agreements list page
2. Done. Should look the same as before.
3. Well, there are some minor stylistic changes made.

## Screenshots

image
image
image

### CODE REVIEW

File: frontend/src/pages/agreements/list/AgreementsList.jsx [hunk 1]
```diff
@@ -172,15 +174,19 @@ export const AgreementsList = () => {
...
-
-            <h1 className="font-sans-lg">Agreements</h1>
-            <p>
-                {myAgreementsUrl
-                    ? "This is a list of the agreements you are listed as a Team Member on."
-                    : "This is a list of all agreements across OPRE."}
-            </p>
-            <AgreementsFilterHeaderSection filters={filters} setFilters={setFilters} />
-            <AgreementsTable agreements={sortedAgreements} />
+            <TablePageLayout
```
Comment (fpigeonjr) [inline frontend/src/pages/agreements/list/AgreementsList.jsx:177]: @johndeange how is this looking so far?

File: frontend/src/components/UI/Layouts/index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# LAYOUTS
```
Comment (fpigeonjr) [inline frontend/src/components/UI/Layouts/index.md:1]: could try this new pattern to document what is inside each directory a la `Nuxt.js` project

Comment (fpigeonjr) [reply frontend/src/components/UI/Layouts/index.md:1]: https://nuxt.com/docs/guide/directory-structure/layouts

File: frontend/src/components/UI/Layouts/TablePageLayout/TablePageLayout.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ * @param {string} props.details - The details to display.
+ * @param {React.ReactNode} props.TabsSection - The tabs to display.
+ * @param {React.ReactNode} props.FilterTags - The filter tags to display.
+ *  @param {React.ReactNode} props.FilterButton - The filter button to display.
+ * @param {React.ReactNode} props.TableSection - The table to display.
+ * @param {string} props.buttonText - The text to display on the button.
+ * @param {string} props.buttonLink - The link to navigate to when the button is clicked.
+ * @returns {React.JSX.Element} - The rendered component.
+ */
+export const TablePageLayout = ({
```
Comment (fpigeonjr) [inline frontend/src/components/UI/Layouts/TablePageLayout/TablePageLayout.js:21]: thought this was a good name for this layout comp

File: frontend/.eslintrc.json [hunk 4]
```diff
@@ -20,7 +20,16 @@
         "react/prop-types": [1, { "ignore": ["children"], "skipUndeclared": true }]
     },
     "plugins": ["react-refresh"],
-    "ignorePatterns": ["**/*.css", "**/*.css.map", "**/*.scss", "**/*.svg", "**/*.png", "**/*.jpg", "**/uswds/**"],
+    "ignorePatterns": [
+        "**/*.md",
```
Comment (fpigeonjr) [inline frontend/.eslintrc.json:24]: no linty on markdown

**Review by johndeange — APPROVED** [review]

**Review by tdonaworth — APPROVED** [review]

**Pull request merged**